client/components/mma/shared/benefits/BenefitsConfiguration.ts

```diff
@@ -1,4 +1,4 @@
-import type { ProductTypeKeys } from '../../../../../shared/productTypes';
+import type { ProductTypeKeys } from '@/shared/productTypes';
 
 /* Product Switch Benefits */
```

Comment:
are people switching to tier 3 or is this comment outdated (or is the code added for T3 unused?)

Reply:
I think it's outdated, these benefits are shown in the product card on the account overview page